kontrakter/varetekt/innsettelsesordre/1.0/innsettelsesordre.schema.json

```diff
@@ -1,297 +1,167 @@
-    "paagrepetSted": {
-      "type": "string",
-      "description": "Sted for pågripelsen."
+    "ordre": {
+      "$ref": "#/definitions/ordre"
     },
     "dokumenter": {
       "type": "array",
-      "items": { "$ref": "#/definitions/dokument" }
+      "description": "PDF versjonen av innsettelsesordren følger med (generert).",
+      "items": {
+        "$ref": "#/definitions/dokument"
+      },
+      "minItems": 1
     }
   },
   "required": [
     "forsendelse",
-    "hovedStraffesaksnummer",
-    "ordreId",
-    "ordreVaretekt",
+    "hovedStraffesak",
+    "bestillingVaretektsplassId",
     "personVaretektInfo",
-    "bestillingAvVaretektsplassId",
-    "paagrepetTidspunkt",
+    "ordre",
     "dokumenter"
   ],
   "additionalProperties": false,
 
   "definitions": {
     "personVaretektInfo": {
-      "title": "personVaretektInfo",
-      "type": "object",
-      "required": ["personVaretekt", "straffesaker"],
-      "additionalProperties": false,
-      "properties": {
-        "personVaretekt": {
-          "$ref": "#/definitions/person"
-        },
-        "forsvarer": { "$ref": "#/definitions/profesjonellPerson" },
-        "varetektSyklusId": {
-          "$ref": "#/definitions/unikId",
-          "description": "Løpenummer nå, UUID senere. Samme identifikator fra pågripelse til du slipper ut av varetekt, se dokumentasjon melding readme.md"
-        },
-        "straffesaker": {
-          "type": "array",
-          "title": "straffesaker",
-          "items": {
-            "$ref": "#/definitions/straffesakInvolverte"
-          },
-          "minItems": 1
-        }
-      }
-    },
-    "profesjonellPerson": {
       "type": "object",
-      "description": "Advokater, tolker, sakkyndige vitner",
+      "description": "Informasjon om siktede person.",
       "properties": {
-        "tittel": {
-          "type": "string"
-        },
-        "navn": {
-          "type": "object",
-          "properties": {
-            "fornavn": {
-              "type": "string"
-            },
-            "mellomnavn": {
-              "type": "string"
-            },
-            "etternavn": {
-              "type": "string"
-            }
-          },
-          "required": ["etternavn"],
-          "additionalProperties": false
+        "person": {
+          "$ref": "#/definitions/siktetPersonEnkel"
         },
-        "kontakt": {
-          "$ref": "#/definitions/kontaktInfo"
-        }
-      },
-      "required": ["navn"],
-      "additionalProperties": false
-    },
-    "straffesakInvolverte": {
-      "type": "object",
-      "description": "Straffesak med statistikkgrupper, krimtype og de involverte",
-      "properties": {
-        "straffesaksnummer": { "$ref": "#/definitions/straffesaksnummer" },
-        "detaljer": { "$ref": "#/definitions/straffesakDetaljer" },
-        "involverte": { "$ref": "#/definitions/involverteStraffesak" }
+        "paagrepetTidspunkt": { "$ref": "#/definitions/datoTid" },
```

Comment:
Hva er tanken bak at paagrepetTidspunkt er her? Den er allerede del av BestillingVaretektsplass-meldingen (personVaretektInfo/paagripelse).

Strukturen her er annerledes enn for eksempel i  BestillingVaretektsplass-meldingen, hvor man har krimineltNettverk og andre informasjonselementer som ikke er i andre meldinger. Vil BestillingVaretektsplass-meldingen eventuelt benyttes for å oppdatere denne informasjonen i forbindelse med at påtale sender Innsettelsesordre-meldingen?

Reply:
Det er nok ikke nødvendig å ha med `paagrepetTidspunkt` her nei, gitt den sendes med bestillingen (men ser den er optional i bestillingen). Eneste jeg kan tenke på der er om personen som sender bestillingen ikke har pågripelsestidspunkt, slik at den heller kommer med innsettelsesordren, men jeg har ikke hørt noe om det. 

Jeg skal høre litt, og sletter evt `paagrepetTidspunkt`

Reply:
Vi har ikke snakket om at vi skal bruke bestillingVaretektsplass-meldingen til å oppdatere informasjon nei. Bruker vil alltid sende bestillingen først, og deretter innsettelsesordren.